apcd-cms/src/apps/admin_regis_table/templates/list_registrations.html

```diff
@@ -6,18 +6,14 @@
 <link rel="stylesheet" href="{% static 'admin_regis_table/registrations_table.css' %}" type="text/css">
 <link rel="stylesheet" href="{% static 'admin_regis_table/registrations_admin_modal.css' %}" type="text/css">
 
-<style>
-  .page-text {
-    margin-top: 27px;
-    margin-bottom: 27px;
-  }
-</style>
-
 <div class="container">
-<h1 class="page-text">List Registrations</h1>
-<hr style="margin-top: 10px">
-<p class="page-text">This is a list of all completed registration forms, they are requesting to become a data submitter.</p>
-<hr style="margin-top: 10px">
+  {% include "nav_cms_breadcrumbs.html" %}
+
+  <h1>List Registrations</h1>
+  <hr />
+  <p style="margin-bottom: 30px">This is a list of all completed registration forms, they are requesting to become a data submitter.</p>
```

Comment:
**Temporary.** This inline style (🤢) is here until I decide how to standardize it[^1] **or** pull it from CMS admin content.

[^1]: More specifically, decide whether `h1 + hr + p + hr { /* negative margin */ }` is "worse" or "better".